li-hadoop-plugin/src/main/resources/dependencyPattern.json

```diff
@@ -5,7 +5,7 @@
       "message": "Open source versions(org.apache.*) of artifacts are not allowed, please use linkedin artifacts(com.linkedin.*). Open source versions will be completely blocked after March 1st 2016",
```

Comment:
The dependency plugin is actually disabled in the hadoop-plugin. We've enabled it only in li-hadoop-plugin, which is a Linkedin specific version of the plugin.